### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by ricarora** [thomaslindstrom/color-picker#65]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (/Users/Tom/Downloads/Atom.app/Contents/Resources/app/src/atom.js:54:11)
Object.activate (/Users/Tom/.atom/packages/color-picker/lib/ColorPicker.coffee:19:21)
```

**Comment by felixkiss** [comment]
Can be closed, because it is already fixed by 85f8815a9fbec8689ed308c035f15e178faa9cc6

**Comment by thomaslindstrom** [comment]
Thanks, @felixkiss.
